styles/LifeAsATorchboxer.module.scss

```diff
@@ -0,0 +1,76 @@
+.container {
+    padding: $grid;
+    display: flex;
+    flex-direction: column;
+
+    @include media-query(medium) {
+        padding: $grid * 2;
+    }
+
+    @include media-query(large) {
+        padding: $grid * 4;
+        flex-direction: row;
+    }
+}
+
+.textContainer {
+    text-align: left;
+    margin-bottom: $grid;
+    order: 1;
+
+    @include media-query(medium) {
+        order: 2;
+    }
+
+    @include media-query(large) {
+        max-width: 635px;
```

Comment:
**Question:** This looks more like 540px in the designs, was there a reason you added this size?

Reply:
This text container encompasses the title (540px) and the description text (635px)